tests/ui/config/karma.conf.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (config) {
             'ui/js/logviewer.js',
             'ui/js/treeherder_app.js',
             'ui/js/controllers/**/*.js',
-            'ui/js/directives/**/*.js',
+            'ui/js/directives/**/**/*.js',
```

Comment:
Change this from `directives/**/**/*.js` -> `directives/treeherder/*.js` and the unit tests should pass.